Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -32,7 +36,7 @@ define([
      *
      * @exception {DeveloperError} Element with id "container" does not exist in the document.
      */
-    function InfoBox(container) {
+    function InfoBox(container, allowScripts) {
```

Comment:
We would normally take an options object here instead for future proofing, but I would just remove this option, since we're not using it anywhere internally and it doesn't add much.

Reply:
I did not want to take an object so as not to break anything.  My first thoughts was not to allow control over `allowScripts` on init at all.  Regarding the arguments vs. object notation, you'd might want to see this: https://jsperf.com/passing-object-vs-passing-raw-value/5 (I've changed the test a bit to our case, but note the first revision which was tested on many browsers).

Reply:
Performance is not a concern in this case because:

1. Construction of objects is slow to begin with, so we try to avoid it whenever possible in Cesium.
2. Most applications will only ever construct a single InfoBox so this is not a hot area of code.

In either case, I would just remove this parameter since we'd like to just have people access the sandbox attributes directly anyway.

Reply:
The problem with just exposing the `sandbox` is that changing the `sandbox` after the `iFrame` is in the DOM should have no effect according to W3C (it's hard to say, but IE and EDGE are correct in this matter, and the others got it "wrong" - https://www.w3.org/TR/2011/WD-html5-20110525/the-iframe-element.html
```
These flags only take effect when the nested browsing context of the iframe is navigated. Removing them, or removing the entire sandbox attribute, has no effect on an already-loaded page.
```
So if we want to allow someone to change the attribute on the fly, we would need to either expose a `setter` or a `method` that does that. What do you think?